sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1718,8 +1718,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
       val message = intercept[AnalysisException] {
         sql("SHOW COLUMNS IN tbl FROM a.b.c")
       }.getMessage
-      assert(message.contains(
-        "Table or view not found: a.b.c.tbl"))
+      assert(message.contains("requires a single-part namespace"))
```

Comment:
Hmm, this could be a little misleading since this sounds like `SHOW COLUMNS IN` is only supported for the session catalog? (although it is currently not supported for v2 catalogs.) This may be OK until we start supporting this command for v2 catalogs.